.github/workflows/superset-build.yaml

```
@@ -0,0 +1,33 @@
+name: Build and Push Superset Image
+run-name: Build superset image ${{ inputs.superset_version }}
+permissions:
+  contents: read
+on:
+  workflow_dispatch:
+    inputs:
+      superset_version:
+        description: "Superset image tag to publish (e.g. 6.1.0)"
+        required: true
+        type: string
```

YAMLlint (1.38.0)

[error] string value is not quoted with any quotes

(quoted-strings)